docs/autoplay.md

```diff
@@ -0,0 +1,82 @@
+## Managing Auto Play
+If you would like to start automatically playing your content, you can easily manipulating the player configuration and set the `playback.autoplay` value accordingly.
+
+>### playback.autoplay
+>##### Type: `boolean`
+>##### Default: `false`
+>##### Description: Start playback automatically.
+>If `true`, playback will start automatically when sources will be given to the player. If `false`, a user gesture will be required to start playback.
+#### Example - Basic usage:
+```js
+var config = {
+  playback: {
+    autoplay: true
+  },
+  sources: {...}
+};
+// No need to add additional code, 
+// playback will start automatically!
+var player = Playkit.loadPlayer(config);
+```
+
+>**_Note_**:
+> Some developers may 
+> simulate the autoplay behavior by calling `play()` method right after the creation of the player without manipulating the player configuration. We strongly recommend **not** doing it this way!
+>```js
+>var config = {
+>  sources: {...}
+>};
+>var player = Playkit.loadPlayer(config);
+>// Bad practice!
+>player.play(); 
+>```
+> #### Why it's bad?
+> When it's done as noted above, the player has no way to know whether the call to `play()` has done via user action or via API call, a fact that can cause unexpected behaviour on browsers which are not support autoplay.
+### What if the browser does not allow autoplay?
+Unfortunately, on some platforms, web browsers does not allow to start playback automatically.
+The platforms that are blocks those attempts are mostly targeted to web browsers on mobile devices, but lately desktop browsers
+ are also start to align those restrictions in order to create unified behavior between platforms (currently Safari 11 and began of Chrome 64).
```

Comment:
On some platforms, web browsers do not allow to playback to begin automatically. The platforms that block attempts at playback are mostly targeted at web browsers on mobile devices; however, some desktop browsers have also started to align these restrictions to create a unified behavior between platforms (currently Safari 11 and beginning with Chrome 64).